src/main/kotlin/minesweeper/model/Board.kt

```diff
@@ -37,4 +37,21 @@ class Board(
     fun getRowCells(row: Int): Cells {
         return cells.getRowCells(row)
     }
+
+    fun openCells(row: Int, column: Int): Boolean {
+        if (isMine(row, column)) {
+            return false
+        }
+
+        cells.openAroundCells(row, column)
+        return true
+    }
+
+    fun isFinishGame(): Boolean {
+        return cells.cellList.all { it.isOpen || it.isMine }
+    }
+
+    private fun isMine(row: Int, column: Int): Boolean {
+        return cells.cellList.find { it.isAround(row, column) }?.isMine ?: false
+    }
```

Comment:
두 함수 모두 cells의 내부 값을 직접 꺼내서 연산을 수행하고 있습니다. 
```suggestion
    fun isFinishGame(): Boolean {
        return cells.allMatch{ it.isOpen || it.isMine }
    }

    private fun isMine(row: Int, column: Int): Boolean {
        return cells.findCell { it.isAround(row, column) }?.isMine ?: false
    }
```

 캡슐화를 유지하는 방식으로 설계해보면 좋을 것 같아요

Reply:
네 더좋은방법이네요!